### PR TITLE
[shuffle] Shuffle Tutorial readme

### PR DESCRIPTION
**Pull request by sunmilee**
## Motivation

Shuffle Tutorial readme first pass 

## Test Plan

N/A

### CODE REVIEW

File: shuffle/cli/README.md [hunk 1]
```diff
@@ -1,70 +1,32 @@
...
-### Extending functionality
-
-1. Freestyle in the typescript REPL console, using [deno libraries](https://deno.land/x)
-1. Modify `/tmp/helloblockchain/main/mod.ts` with more functions
-1. Run E2E tests against your custom genesis with `shuffle test e2e`
 
-## Development
+## Commands Overview
+1. `shuffle new`: Creates a new shuffle project for Move development
+2. `shuffle nonde`: Runs a local devnet
```
Comment (avin1100) [inline shuffle/cli/README.md:22]: typo

File: shuffle/cli/README.md [hunk 2]
```diff
@@ -1,70 +1,32 @@
...
```
Comment (avin1100) [inline shuffle/cli/README.md:22]: change to shuffle node

File: shuffle/cli/tutorials/HelloBlockchain.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Skipping Module: 00000000000000000000000000000001::Errors
+Skipping Module: 00000000000000000000000000000001::BitVector
+Skipping Module: 00000000000000000000000000000001::Capability
+Skipping Module: 00000000000000000000000000000001::FixedPoint32
+Skipping Module: 00000000000000000000000000000001::Option
+Skipping Module: 00000000000000000000000000000001::BCS
+Skipping Module: 00000000000000000000000000000001::GUID
+Skipping Module: 00000000000000000000000000000001::Event
+Deploying Module: 24163AFCC6E33B0A9473852E18327FA9::Message
+```
```
Comment (avin1100) [inline shuffle/cli/tutorials/HelloBlockchain.md:94]: Did we purposefully leave the NFT/TestNFTs out of this output?

File: shuffle/cli/tutorials/HelloBlockchain.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+## Explore with Console
+
+Once you have your module deployed, you can use `shuffle console` to enter a typescript REPL.
+Enter the REPL inside your project folder and try running the below commands:
+
+```
+// Set the message field inside MessageHolder resource of your account to "hello blockchain"
+await main.setMessageScriptFunction("hello blockchain", await Shuffle.sequenceNumber());
```
Comment (avin1100) [inline shuffle/cli/tutorials/HelloBlockchain.md:103]: This is outdated. change to await main.setMessageScriptFunction("hello blockchain")

File: shuffle/cli/tutorials/HelloBlockchain.md (hunk 3 above)
Comment (sunmilee) [reply shuffle/cli/tutorials/HelloBlockchain.md:94]: yep

File: shuffle/cli/tutorials/HelloBlockchain.md (hunk 4 above)
Comment (sunmilee) [reply shuffle/cli/tutorials/HelloBlockchain.md:103]: good catch

**Review by avin1100 — APPROVED** [review]

**Review by xli — APPROVED** [review]

**Review by xli — DISMISSED** [review]

**Pull request closed without merging**